test/integration/preflight/check-yum-update/test-yum-repo-broken.yml

```diff
@@ -0,0 +1,16 @@
+- hosts: localhost
+  tasks:
+
+  - include: tasks/create-test-container.yml
+
+  - name: break the local yum repo
+    delegate_to: dockertest
+    replace: dest=/etc/yum.repos.d/local.repo backup=no
+             regexp='^baseurl'
+             replace='#baseurl'
```

Comment:
Building images is relatively expensive while running simple commands at container start isn't... since we don't have to ship images, we can build all we want, but on the other hand if we stuff everything into one image and reuse it for multiple tests, that will save on the expense of building. We can just use our judgment on each case.